package/rdspgbadger.py

```diff
@@ -131,21 +133,22 @@ def write_log(client, dbinstance_id, filename, logfilename):
                     raise
         with open(filename, "a") as logfile:
             if 'LogFileData' in response:
-                downloaded_lines = response["LogFileData"].count("\n")
-                if (response["AdditionalDataPending"] and
-                        downloaded_lines < max_number_of_lines):
+                if truncated_string in response["LogFileData"][-slice_length:]:
```

Comment:
Any particular reason not testing with `==` here?

Reply:
It won't work as the data ends with `\n`. One could use the slice expression `[-slice_length:-1]` or even add `.strip("\n")` to make the `==` comparison work, but I thought it looked simpler that way.